packages/quantic/force-app/main/default/lwc/quanticNumberButton/quanticNumberButton.js

```diff
@@ -1,18 +1,34 @@
 import {LightningElement, api} from 'lwc';
 
+/**
+ * The `QuanticNumberButton` component is used internally to display a button in a set of buttons with numeric labels.
+ * @fires CustomEvent#goto
+ * @example
+ * <c-quantic-number-button number="1" current="1" ongoto={goto}></c-quantic-number-button>
+ */
 export default class QuanticNumberButton extends LightningElement {
+  /**
+   * The number to display as button label.
+   * @api
+   * @type {number}
+   */
   @api number;
+  /**
+   * The selected number in the set of buttons.
+   * @api
+   * @type {number}
+   */
   @api current;
```

Comment:
I feel it would make more sense to have a `selected` property instead of `current`.

Reply:
sure